x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/dataframe/DataFrameAnalyticsManager.java

```diff
@@ -147,7 +158,8 @@ private void startAnalytics(DataFrameAnalyticsTask task, DataFrameAnalyticsConfi
         DataFrameDataExtractorFactory.create(client, Collections.emptyMap(), config, dataExtractorFactoryListener);
```

Comment:
I think we can remove the headers argument here and read it straight from the config

Reply:
Mos def, this was an oversight